dashboard/config/locales/en.yml

```diff
@@ -327,6 +327,8 @@ en:
     delete_text_teacher_note: "Please note: deleting your account will not delete your students' accounts."
     delete_confirm: 'Are you sure you want to delete your account?'
     deleted_user: 'deleted user'
+    no_password_because_oauth: 'No password because you sign in with a third-party provider.'
+    no_password_because_sponsored: 'See your teacher to manage your password.'
```

Comment:
@tanyaparker do I need to do anything other than edit this file when adding new i18n strings to dashboard?